core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -466,6 +473,9 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
       case USAGE_ERROR =>
         printUsageAndExit(1)
 
+      case KUBERNETES_NAMESPACE =>
```

Comment:
Nit. I would expect a significant parameter like this to come before line 464. Line 464 - 473 are mostly about help and usage errors, it seems.

Reply:
Done.